services/workflows-service/src/collection-flow/dto/update-flow-input.dto.ts

```diff
@@ -87,34 +88,21 @@ export class UBOShareholder {
 }
 
 export class UpdateFlowPayload {
-  @ValidateNested()
-  @Type(() => MainRepresentative)
-  mainRepresentative!: MainRepresentative;
-
-  @ValidateNested()
-  @IsArray()
-  @ValidateNested({ each: true })
-  @Type(() => CompanyDocument)
-  documents!: CompanyDocument[];
-
-  @IsArray()
-  @ValidateNested({ each: true })
-  ubos!: UBOShareholder[];
-
   @IsObject()
-  dynamicData!: object;
+  business?: object;
 
-  @IsNullable()
   @IsString()
-  flowState!: string | null;
+  ballerineEntityId?: string;
+
+  @IsObject()
+  endUser?: object;
 
   @IsObject()
-  businessData!: object;
+  context!: object;
```

Comment:
Consider using some broad `business` and `endUser` dtos. Otherwise `Record<string, unknown>` would be easier to work with.